changelog/v3.13.0.md

```diff
@@ -0,0 +1,108 @@
+# v3.13.0
+
+## Summary
+
+The v3.13.0 release includes features and fixes from 67 pull requests since the v3.12.1 release.  New features and improvements include:
+
+* Improved tiles rendering for the canvas renderer
+* Improved MapQuest rendering
+* Add color option to ol.style.Icon
+* Load TileJSON sources via XmlHttpRequest by default
+* Add new ol.geom.LineString#getCoordinateAt function
+* Simplify meters per unit handling
+* Use ESLint as a replacement for gjslint.py and jshint
+
+## Upgrade notes
+
+#### `proj4js` integration
+
+Before this release, OpenLayers depended on the global proj4 namespace. When using for instance browserify you might not want to depend on the global `proj4` namespace. You can use the `ol.proj.setProj4` function to set the proj4 function object. For example in a browserify ES6 environment:
```

Comment:
The second sentence could say "When using a module loader like Browserify, you might ..."